Cake.Recipe/Content/gitversion.cake

```diff
@@ -73,7 +73,7 @@ public class BuildVersion
             semVersion = assertedVersions.LegacySemVerPadded;
             informationalVersion = assertedVersions.InformationalVersion;
             assemblySemVer = assertedVersions.AssemblySemVer;
-            milestone = string.Concat(version);
+            milestone = assertedVersion.SemVer;
```

Comment:
missing an `s`
```suggestion
            milestone = assertedVersions.SemVer;
```

Reply:
Apologies, never saw this suggestion before I noticed this myself.  Change is already in place.